data/datasets.json

```diff
@@ -0,0 +1 @@
+{"s82_rrlyrae": {"description": "This dataset contains 483 RR Lyrae from Sesar 2010. This dataset was sourced from astroml, and then massaged into the parquet format for ingest into TAPE.", "source_file": "https://github.com/lincc-frameworks/tape_benchmarking/blob/main/data/s82_rrlyrae/s82rrl_source.parquet?raw=True", "object_file": "https://github.com/lincc-frameworks/tape_benchmarking/blob/main/data/s82_rrlyrae/s82rrl_object.parquet?raw=True"}, "s82_qso": {"description": "This dataset contains recalibrated light curves for all spectroscopically confirmed QSOs in the SDSS DR7 stripe 82 (22h 24m < R.A. < 04h 08m and | Dec | < 1.27 deg, about 290 deg2). The total number of QSOs is 9,258, and the observations are spaced out over ~10 years in yearly 'seasons' about 2-3 months long.. This dataset was sourced from http://faculty.washington.edu/ivezic/cmacleod/qso_dr7/Southern.html, and then massaged into the parquet format for ingest into TAPE.", "source_file": "https://github.com/lincc-frameworks/tape_benchmarking/blob/main/data/s82_qso/s82qso_source.parquet?raw=True", "object_file": "https://github.com/lincc-frameworks/tape_benchmarking/blob/main/data/s82_qso/s82qso_object.parquet?raw=True"}}
```

Comment:
I would suggest replacing "massaged" with something like "reformatted" as it might be easier for people to understand.